pkgs/development/tools/dive/default.nix

```diff
@@ -25,6 +24,7 @@ buildGoModule rec {
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = lib.optionals stdenv.isLinux [ btrfs-progs gpgme lvm2 ];
+  patches = [ ./scrolling.patch ];
```

Comment:
We cannot fetch the individual commits from https://github.com/wagoodman/dive/pull/478/commits ?

Reply:
Sure, will update my PR.